geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -7,27 +7,138 @@ Version 2.6.0
+The goal of this change is to move all thies configurationthe the `vars.yaml` file (with default values in
+the `CONST_vars.yaml`). This will make the future upgrade easyer and incrase the configuration possibility
+in the simple application mode, see the documentation
+<https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/create_application.html#simple-application>.
+
+
+When you apply the `ngeo.diff` you should first apply the changes in the `geoportal/vars.yaml` file.
+
+For the next you should consultate the documentation about:
+- The interfaces_config documentation
+<https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/ngeo.html?highlight=ngeo#dynamic-json-view>
+- GMF constants definitions <https://camptocamp.github.io/ngeo/master/jsdoc/module-contribs_gmf_src_options.html>
+- ngeo constants definitions <https://camptocamp.github.io/ngeo/master/jsdoc/module-src_options.html>
+
+The 'srid', 'alternate_projections', 'resolutions' and 'extent' `add path` will be configure on one point and will be
+dispatch using c2ctemplate or YAML link.
+
+Kind of thinks you can see:
+
+Example in API
+..............
+
+In the ngeo.diff:
+```
+--- a/geoportal/cartoriviera_geoportal/static-ngeo/api/index.js
++++ b/geoportal/cartoriviera_geoportal/static-ngeo/api/index.js
+...
+-// The URL to the search service.
+-config.searchUrl = '{FULL_ENTRY_POINT}search?interface=api&limit=15';
+```
+And in your `geoportal/cartoriviera_geoportal/static-ngeo/api/index.js` file:
+```
+// The URL to the search service.
+onfig.searchUrl = '{FULL_ENTRY_POINT}search?interface=api&limit=15';
+```
+We see that that your settings didn't corresponds to the previus default settings, than you should have in
+you vars file somthing like:
+```
+vars:
+  interfaces_config:
+    api:
+      routes:
+        searchUrl:
+          params:
+            limit: 15
+
+update_paths:
+  - interfaces_config.api.routes.searchUrl  # This one sould already be present
+```
+
+
+Example in controller constructor
+.................................
+
+In the ngeo.diff:
+```
+-  constructor($scope, $injector) {
+-    super({
+-      srid: 2056,
+-      mapViewConfig: {
+-        center: [2632464, 1185457],
+-        zoom: 3,
+-        resolutions: [250, 100, 50, 20, 10, 5, 2, 1, 0.5, 0.25, 0.1, 0.05],
+-        constrainResolution: true,
+-        extent: [2485071.54, 175346.36, 2828515.78, 1299941.84],
+-      }
+-    }, $scope, $injector);
+```
+And in your `geoportal/cartoriviera_geoportal/static-ngeo/api/index.js` file:
+```
+  constructor($scope, $injector) {
+    super({
+      maxTilesLoading: Infinity,
+      srid: 2056,
+      mapViewConfig: {
+        center: [2559045, 1144195],
+        zoom: 1,
+        constrainResolution: true,
+        resolutions: [50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5, 0.25, 0.1, 0.05, 0.025]
+      }
+    }, $scope, $injector);
+```
+We see that that your settings didn't corresponds to the previus default settings, then you should have in
```

Comment:
We see that the project settings didn't correspond to the previous default settings, therefore you should have in
your vars file something like: